tests/performance/test_footprint.py

```diff
@@ -1,77 +1,79 @@
 import platform
 import time
-from typing import AsyncIterator, cast
+from typing import AsyncIterator
 
 import aiomcache
 import pytest
-import redis.asyncio as redis
+from glide import GlideClient, GlideClientConfiguration, NodeAddress
 
 
 @pytest.fixture
-async def redis_client() -> AsyncIterator["redis.Redis[str]"]:
-    async with cast("redis.Redis[str]",
-                    redis.Redis(host="127.0.0.1", port=6379, max_connections=1)) as r:
-        yield r
+async def valkey_client() -> AsyncIterator["GlideClient"]:
+    addresses = [NodeAddress("localhost", 6379)]
+    conf = GlideClientConfiguration(addresses=addresses)
+    client = await GlideClient.create(conf)
+
+    yield client
```

Comment:
I think this is breaking the tests, doesn't look like we're closing it?
```suggestion
    async with GlideClient.create(conf) as client:
        yield client
```